moped-database/migrations/1668369420328_delete_phase_id_0/down.sql

```diff
@@ -0,0 +1,2 @@
+-- Could not auto-generate a down migration.
+SELECT 0;
```

Comment:
I'm 100% cool with not completing a perfect reconstruction of the state before the up ran. I really don't see us needing the migration rollback capability, esp not this far in. And I am even more confident that we wouldn't be able to really use what we have historically without a reset-and-rebase type of start-over operation. This is 👍.

Reply:
👍 thanks for that!